From db1990f92c12b6a2f83632338cd7a512b8fc5f7c Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Mon, 6 Aug 2018 17:34:27 +0800
Subject: [PATCH 085/107] arm: layerscape: Remove CONFIG_U_QE define in ls1012atwr.h

Move to drivers/qe/Kconfig. QE has problem in LS1021A deep sleep
resume, so make this option in menuconfig instead.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
---
 include/configs/ls1021atwr.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/include/configs/ls1021atwr.h b/include/configs/ls1021atwr.h
index 8f27d9e..548e0c5 100644
--- a/include/configs/ls1021atwr.h
+++ b/include/configs/ls1021atwr.h
@@ -108,7 +108,6 @@
 
 #if !defined(CONFIG_SD_BOOT) && !defined(CONFIG_NAND_BOOT) && \
 	!defined(CONFIG_QSPI_BOOT)
-#define CONFIG_U_QE
 #define CONFIG_SYS_QE_FMAN_FW_IN_NOR
 #endif
 
-- 
1.7.1

